Src/Plugins/Security/SEC101_004.FacebookAppCredentialsValidator.cs

```diff
@@ -45,7 +53,7 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
             string secret = fingerprint.Secret;
 
             ValidationState state = RetrieveInformation(
-                $"https://graph.facebook.com/oauth/access_token?client_id={id}&client_secret={secret}&grant_type=client_credentials",
+                string.Format(OAuthUri, id, secret),
```

Comment:
Please consider adding CultureInfo to string.Format to avoid unexpected behavior.